### PR TITLE
Adds the Courage from Voidcrew.

### PR DESCRIPTION
**Pull request by ForestCranked**
## About The Pull Request

This ports a slightly tweaked version of the Courage from Voidcrew to Shiptest.

## Why It's Good For The Game

I don't have anything to really put here, as I started tweaking the Courage on a whim and ended up with something decent enough to make me think "I should try and actually get this into Shiptest." I won't be surprised if this gets denied.

## Changelog
:cl:
add: The Courage's .dmm file.
add: The Courage's config .json file.
/:cl:

image

### CODE REVIEW

**Comment by rye-rice** [comment]
I said it in the discord, but I'm against porting voidcrew ships because they are usually low quality and have bizarre balancing decisions.  This one is literally the caravan ruin syndie ship with stuff added to it

File: _maps/configs/SYN-C_courage-VOID.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+	"map_name": "Courage-class Strike Dropship",
+	"map_short_name": "Courage-class",
+	"prefix": "SYN-C",
```
Comment (Apogee-dev) [inline _maps/configs/SYN-C_courage-VOID.json]: “prefix”: “SSV”,
“namelists”: [“SPACE”, “WEAPONS”, “NATURAL_AGGRESSIVE”, “BEASTS”]

Prefix should follow established schemes for factions (SSV for syndicate), namelists suggested based on other syndicate ships

File: _maps/configs/SYN-C_courage-VOID.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	"map_path": "_maps/shuttles/shiptest/courage.dmm",
+	"map_id": "courage_class",
+	"job_slots": {
```
Comment (Apogee-dev) [inline _maps/configs/SYN-C_courage-VOID.json:9]: “limit”: 1,

If you’re going to attempt to get a ship this strong merged at least put a cap on how many can spawn.

**Review by Apogee-dev — CHANGES_REQUESTED** [review]

**Comment by Apogee-dev** [comment]
Please add a preview image of the ship.

File: _maps/configs/SYN-C_courage-VOID.json (hunk 2 above)
Comment (ForestCranked) [reply _maps/configs/SYN-C_courage-VOID.json:9]: Good to know, sorry for not including that.

**Review by Apogee-dev — APPROVED** [review]

File: _maps/configs/SYN-C_courage-VOID.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+	"map_name": "Courage-class Strike Dropship",
+	"map_short_name": "Courage-class",
+	"prefix": "SSV",
+	"namelists": [“SPACE”, “WEAPONS”, “NATURAL_AGGRESSIVE”, “BEASTS”],
```
Comment (ZephyrTFA) [inline _maps/configs/SYN-C_courage-VOID.json]:
```suggestion
	"namelists": ["SPACE", "WEAPONS", "NATURAL_AGGRESSIVE", "BEASTS"],
```

**Comment by Apogee-dev** [comment]
Now that there’s a map preview, I’m not sure what this brings to the table that other ships don’t do in a more fun package. The ship itself doesn’t seem to have any features other than the combat gear. Most other combat ships offer at least a little room for RP or building (for example, the hyena’s crew areas and cargobay, or the carina’s briefing room and hangar)

**Comment by ZephyrTFA** [comment]
This ship doesn;t bring anything to the table except blaring "I'm a pirateship"
Unless you can modify this ship to not be about violence and more violence it doesn't have a place on our shiproster.

**Pull request closed without merging**